### PR TITLE
running the helm release script and adding an env folder for demo

### PR DESCRIPTION
**Pull request by scott-robertson1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-43

### Change description ###
running the add-namespace-to-env and the add-helm-release scripts for demo

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/darts-modernisation/darts-api/darts-api.yaml [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
       sourceRef:
         kind: GitRepository
         name: hmcts-charts
         namespace: flux-system
-      interval: 8m
+      interval: 1m
+  values:
+    java:
+      image: sdshmctspublicsbox.azurecr.io/darts/api:latest # {"$imagepolicy": "flux-system:darts-api"}
+      ingressHost: darts-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```
Comment (timja) [inline apps/darts-modernisation/darts-api/darts-api.yaml:20]: this shouldn't be set here, I've fixed in the script: https://github.com/hmcts/sds-flux-config/pull/2817
```suggestion
```

File: apps/darts-modernisation/darts-api/darts-api.yaml [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+      disableTraefikTls: true
+    global:
+      environment: demo
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
```
Comment (timja) [inline apps/darts-modernisation/darts-api/darts-api.yaml:24]: This shouldn't be set here:
https://github.com/hmcts/sds-flux-config/pull/2818
```suggestion
```

File: apps/darts-modernisation/darts-portal/darts-portal.yaml [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
       sourceRef:
         kind: GitRepository
         name: hmcts-charts
         namespace: flux-system
-      interval: 8m
+      interval: 1m
+  values:
+    nodejs:
+      image: sdshmctspublicsbox.azurecr.io/darts/portal:latest # {"$imagepolicy": "flux-system:darts-portal"}
+      ingressHost: darts-portal-{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (timja) [inline apps/darts-modernisation/darts-portal/darts-portal.yaml:20]: https://github.com/hmcts/sds-flux-config/pull/2817
```suggestion
```

File: apps/darts-modernisation/darts-portal/darts-portal.yaml [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
+      disableTraefikTls: true
+    global:
+      environment: demo
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
```
Comment (timja) [inline apps/darts-modernisation/darts-portal/darts-portal.yaml:24]: https://github.com/hmcts/sds-flux-config/pull/2818
```suggestion
```

**Review by timja — CHANGES_REQUESTED** [review]

**Review by timja — APPROVED** [review]

**Review by madjava — APPROVED** [review]

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**